### PR TITLE
Add more documentation and an example

### PR DESCRIPTION
**Pull request by carlossg**
Move options under Advanced tab
Rename plugin to CloudBees Docker Build and Publish to avoid confusion with other Docker related projects

### CODE REVIEW

**Comment by carlossg** [comment]
@reviewbybees

**Comment by kohsuke** [comment]
:+1: 

We should also get +1 from @michaelneale, the original author of this plugin.

**Comment by michaelneale** [comment]
Taking a look today. Got some other PRs to merge and fixes today.

File: src/main/java/com/cloudbees/dockerpublish/DockerBuilder.java [hunk 1]
```diff
@@ -332,7 +332,7 @@ public boolean isApplicable(Class<? extends AbstractProject> aClass) {
          * This human readable name is used in the configuration screen.
          */
         public String getDisplayName() {
-            return "Docker build and publish";
+            return "CloudBees Docker Build and Publish";
```
Comment (ndeloof) [inline src/main/java/com/cloudbees/dockerpublish/DockerBuilder.java]: Do we really want this to be the build step name ? I mean, will all plugin author prepend UI labels with his company name ?

Comment (michaelneale) [inline src/main/java/com/cloudbees/dockerpublish/DockerBuilder.java]: Agree - this should just say docker...

Comment (rsandell) [inline src/main/java/com/cloudbees/dockerpublish/DockerBuilder.java]: I think it should also be put into a ```Messages.properties```

Comment (carlossg) [inline src/main/java/com/cloudbees/dockerpublish/DockerBuilder.java]: updated the PR removing CloudBees from the displayName

Comment (carlossg) [inline src/main/java/com/cloudbees/dockerpublish/DockerBuilder.java]: actually just pushed to PR #14

**Pull request merged**